package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harnessio/ff-nodejs-server-sdk",
-  "version": "1.2.5",
+  "version": "1.2.6",
```

Comment:
Looks like `package-lock.json` doesn't have this version. You probably just ned to run `npm i`